Common/Log.h

```diff
@@ -89,7 +89,7 @@ bool GenericLogEnabled(LogLevel level, Log type);
 #if defined(_DEBUG) || (!PPSSPP_PLATFORM(ANDROID) && !PPSSPP_PLATFORM(IOS))
 
 // Needs to be an int (and not use the enum) because it's used by the preprocessor!
-#define MAX_LOGLEVEL DEBUG_LEVEL
+#define MAX_LOGLEVEL VERBOSE_LEVEL
```

Comment:
I might consider changing this to VERBOSE_LEVEL for debug builds only, but not for release builds without doing an evaulation of the performance impact.

Reply:
Does the MAX_LOGLEVEL set to Verbose affects the performance even when the Log channels are set to Info by end user?